RELEASE.md

```diff
@@ -11,7 +11,7 @@ Release cadence of first pre-releases being cut is 6 weeks.
 | v2.4           | 2018-09-06                                 | Goutham Veeramachaneni (GitHub: @gouthamve) |
 | v2.5           | 2018-10-24                                 | Frederic Branczyk (GitHub: @brancz)         |
 | v2.6           | 2018-12-05                                 | Simon Pasquier (GitHub: @simonpasquier)     |
-| v2.7           | 2019-01-16                                 | **searching for volunteer**                 |
+| v2.7           | 2019-01-16                                 | Goutham Veeramachaneni (GitHub: @gouthamve) |
```

Comment:
We should probably fill in the next few dates too

Reply:
Done!